src/stores/toasts.ts

```diff
@@ -1,17 +1,24 @@
 import { writable } from "svelte/store";
 
-export const errorToasts = writable<{ id: number; message: string }[]>([]);
+export const toasts = writable<
+    { id: number; typ: "error" | "info"; message: string }[]
+>([]);
 
 let nextId = 0;
 
 /**
- * Show an error toast
- * @param message The error message
+ * Show an toast
+ * @param message The message
+ * @param typ Which typ of toast should be shown, either error or info
  * @param timeout Timeout in milliseconds after which to remove the toast
  */
-export function showErrorToast(message: string, timeout = 8000): void {
+export function showToast(
+    message: string,
+    typ: "error" | "info",
```

Comment:
```suggestion
    type: "error" | "info",
```

Also rename it in the store please.